### ISSUE TITLE
Wrong result when parsing large maps

### ISSUE THREAD

**Issue by Illviljan** [dkuschmierz/dcmReader#1]
When lines in the dcm file are too long the values are wrapped to a new line for better readability. 
dcmReader appears to overwrite the values rather than appending them?

```python
import dcmReader 
dcmReader.dcmReader.logger.disabled = False
d = r"sample_map.DCM"
dcm = dcmReader.DcmReader()
dcm.read(d)

m = dcm.getGroupCharacteristicMaps()[0]
shape = (m.xDimension, m.yDimension) 

# Correct length here:
print(m.name, shape)
Test_w (16, 15)

# m.values appears to only take the last row:
m.values
Out[157]: 
{0.0: {1225.0: 10.0, 1325.0: 10.0, 1425.0: 10.0, 1525.0: 10.0},
 50.0: {1225.0: 10.0, 1325.0: 10.0, 1425.0: 10.0, 1525.0: 10.0},
 100.0: {1225.0: 10.0, 1325.0: 10.0, 1425.0: 10.0, 1525.0: 10.0},
 150.0: {1225.0: 10.0, 1325.0: 10.0, 1425.0: 10.0, 1525.0: 10.0},
 200.0: {1225.0: 10.0, 1325.0: 10.0, 1425.0: 10.0, 1525.0: 10.0},
 250.0: {1225.0: 10.0, 1325.0: 10.0, 1425.0: 10.0, 1525.0: 10.0},
 300.0: {1225.0: 10.0, 1325.0: 10.0, 1425.0: 10.0, 1525.0: 10.0},
 350.0: {1225.0: 10.0, 1325.0: 10.0, 1425.0: 10.0, 1525.0: 10.0},
 400.0: {1225.0: 10.0, 1325.0: 10.0, 1425.0: 10.0, 1525.0: 10.0},
 450.0: {1225.0: 10.0, 1325.0: 10.0, 1425.0: 10.0, 1525.0: 10.0},
 500.0: {1225.0: 10.0, 1325.0: 10.0, 1425.0: 10.0, 1525.0: 10.0},
 550.0: {1225.0: 10.0, 1325.0: 10.0, 1425.0: 10.0, 1525.0: 10.0},
 600.0: {1225.0: 10.0, 1325.0: 10.0, 1425.0: 10.0, 1525.0: 10.0},
 650.0: {1225.0: 10.0, 1325.0: 10.0, 1425.0: 10.0, 1525.0: 10.0},
 700.0: {1225.0: 10.0, 1325.0: 10.0, 1425.0: 10.0, 1525.0: 10.0}}
```

Sample data, sample_map.dcm:
<details>

```
* encoding="ISO-8859-1"
* DAMOS format
* Created by CDM V7.3.7 Build 110/1
* Creation date: 2022-12-05
*

KONSERVIERUNG_FORMAT 2.0




FUNKTIONEN
   FKT Test "" ""
END

STUETZSTELLENVERTEILUNG Test_y 15
*SST
   LANGNAME "Test description for y axis"
   FUNKTION Test
   EINHEIT_X "m"
   ST/X   0.0000000000000000   50.0000000000000000   100.0000000000000000   150.0000000000000000   200.0000000000000000   250.0000000000000000
   ST/X   300.0000000000000000   350.0000000000000000   400.0000000000000000   450.0000000000000000   500.0000000000000000   550.0000000000000000
   ST/X   600.0000000000000000   650.0000000000000000   700.0000000000000000
END

STUETZSTELLENVERTEILUNG Test_x 16
*SST
   LANGNAME "Test description for x axis"
   FUNKTION Test
   EINHEIT_X "kg"
   ST/X   25.0000000000000000   125.0000000000000000   225.0000000000000000   325.0000000000000000   425.0000000000000000   525.0000000000000000
   ST/X   625.0000000000000000   725.0000000000000000   825.0000000000000000   925.0000000000000000   1025.0000000000000000   1125.0000000000000000
   ST/X   1225.0000000000000000   1325.0000000000000000   1425.0000000000000000   1525.0000000000000000
END

GRUPPENKENNFELD Test_w 16 15
   LANGNAME "Test description for the map"
   FUNKTION Test
   EINHEIT_X "m"
   EINHEIT_Y "kg"
   EINHEIT_W "-"
*SSTX	Test_x
*SSTY	Test_y
   ST/X   25.0000000000000000   125.0000000000000000   225.0000000000000000   325.0000000000000000   425.0000000000000000   525.0000000000000000
   ST/X   625.0000000000000000   725.0000000000000000   825.0000000000000000   925.0000000000000000   1025.0000000000000000   1125.0000000000000000
   ST/X   1225.0000000000000000   1325.0000000000000000   1425.0000000000000000   1525.0000000000000000
   ST/Y   0.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   ST/Y   50.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   ST/Y   100.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   ST/Y   150.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   ST/Y   200.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   ST/Y   250.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   ST/Y   300.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   ST/Y   350.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   ST/Y   400.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   ST/Y   450.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   ST/Y   500.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   ST/Y   550.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   ST/Y   600.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   ST/Y   650.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   ST/Y   700.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
   WERT   10.0000000000000000   10.0000000000000000   10.0000000000000000   10.0000000000000000
END



```

</details>

**Comment by dkuschmierz** [comment]
Indeed the parser did not support multiline values. Was not aware that this is allowed in the standard.
Just published version 0.1.3 which supports the multiline values. Also added some test cases...

**Comment by Illviljan** [comment]
Nice! Just tried it out and seems to work as expected.
